sdk/python/feast/infra/utils/snowflake_udfs.py

```diff
@@ -0,0 +1,54 @@
+import pandas
+from _snowflake import vectorized
+
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
```

Comment:
@adchia @achals 

How do I get this file to ignore the __init__.py folder in feast main?

This file is the main file where the python udf handler is set, but obviously with wanting to reuse code already written, it looks to import the entire dependecies of the feast project. If I delete the __init__.py file, it works!

Maybe for right now on deployment, I just delete the file before it gets uploaded to snowflake.

Reply:
I'm not quite sure I understand the problem here - how is this file being packaged and used by snowflake? And is there a specific import that's causing other dependencies to come in transitively?